api/src/main/java/org/openmrs/PersonName.java

```diff
@@ -531,21 +539,33 @@ public String toString() {
 	 * @should return negative if other familynamePrefix is greater
 	 * @should return negative if other familyNameSuffix is greater
 	 * @should return negative if other dateCreated is greater
-	 * @deprecated since 1.12. Use DefaultComparator instead.
-	 * Note: this comparator imposes orderings that are inconsistent with equals.
 	 */
-	@SuppressWarnings("squid:S1210")
 	public int compareTo(PersonName other) {
```

Comment:
Same here, you don't have to change the code, just remove the deprecate annotation